app/controllers/trial_scorings_controller.rb

```diff
@@ -19,6 +19,7 @@ def show
           logger.info 'MISS MISS MISS'
           prepare_grid_data.to_json
         end
+        # grid_data = prepare_grid_data.to_json
```

Comment:
Consider adding a comment which explains why this replacement is useful for development+testing.